data/data/agent/files/usr/local/bin/install-status.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+
+inactive_services() {
+    local services="assisted-service.service create-cluster-and-infraenv.service apply-host-config.service start-cluster-installation.service"
+    for s in ${services}; do
+        if ! systemctl is-active "${s}" >/dev/null; then
+            printf "%s " "${s}"
+        fi
+    done
+}
+
+check_services() {
+    local services_issue="/etc/issue.d/70_agent-services.issue"
+    not_started="$(inactive_services)"
+    if [ -z "${not_started}" ]; then
+        rm -f "${services_issue}"
+    else
+        read -ra show_services <<<"${not_started}"
+        {
+            printf '\\e{cyan}Waiting for services:\\e{reset}\n'
+            systemctl --no-legend list-units --all "${show_services[@]}" | awk '{printf("[\\e{cyan}%s\\e{reset}]", $4); for (i=5; i<=NF; i++) {if (i>5 || $i != "start") printf(" %s", $i)}; print ""}'
+        } >"${services_issue}"
+    fi
+}
+
+check_host_config() {
+    local host_config_issue="/etc/issue.d/80_host-config.issue"
+    if [ -f /var/run/agent-installer/host-config-failures ]; then
+        printf '\n\\e{lightred}Installation cannot proceed:\\e{reset}\n%s\n' "$(cat /var/run/agent-installer/host-config-failures)" >"${host_config_issue}"
```

Comment:
I think probably should change "Installation cannot proceed" to "Installation validation in progress" or something. I tested this with a valid installation and this message pops up for about 30 seconds which would leave users to think their install failed, and maybe force them to abort, when in fact the next screen that pops up shows  that it has succeeded/

Reply:
Good point, it's highly ambiguous between "cannot proceed yet" or "will not proceed ever/until you do something".
I suspect it might not have been this one you saw though, but one of the ones in start-cluster-installation.sh. Can you confirm the whole message?

Reply:
You're right, its the one in start-cluster-installation.sh. I've attached a screenshot. So this one can stay, the other one we should indicate that it "may continue" or something, as opposed to "cannot proceed"
![Screenshot from 2022-12-05 12-27-56](https://user-images.githubusercontent.com/19653316/205734687-3bd6b6b6-8ad9-46c7-85b6-134de62a41be.png)

Reply:
Perhaps just change the message to "Installation waiting for hosts" so its clear that it hasn't failed yet